recon/modules/v1_osint/all_modules.py

```
@@ -0,0 +1,438 @@
+    category = "OSINT"
+    description = "Fetch basic GitHub user profile by username."
+
+    def examples(self):
+        return [{"username": "torvalds"}]
+
+    def option_spec(self) -> List[Option]:
+        return [
+            Option("username", required=True, help="GitHub username, e.g. torvalds"),
+        ]
+
+    def run(self, username: str, http: HTTPClient, **kwargs) -> Dict[str, Any]:
```

Ruff (0.13.2)

Unused method argument: `kwargs`

(ARG002)